tests/unit/bcsHelper.test.ts

```diff
@@ -593,4 +593,70 @@ describe("Tests for the Serializable class", () => {
       expect(() => MoveVector.U8([BigInt(1)] as any)).toThrow();
     });
   });
+
+  describe("toSimpleValue tests", () => {
+    it("correctly converts primitive types to inner simple types", () => {
+      const u8 = new U8(1);
+      const u16 = new U16(1);
+      const u32 = new U32(1);
+      const u64 = new U64(1);
+      const u128 = new U128(1);
+      const u256 = new U256(1);
+      const bool = new Bool(true);
+      const address = AccountAddress.fromHexInputRelaxed("0x01020304");
+      const str = new MoveString("This is a string. It's a simple string...");
+      expect(u8.toSimpleValue()).toEqual(1);
+      expect(u16.toSimpleValue()).toEqual(1);
+      expect(u32.toSimpleValue()).toEqual(1);
+      expect(u64.toSimpleValue()).toEqual(1n);
+      expect(u128.toSimpleValue()).toEqual(1n);
+      expect(u256.toSimpleValue()).toEqual(1n);
+      expect(bool.toSimpleValue()).toEqual(true);
+      expect(address.toSimpleValue()).toEqual(
+        new Uint8Array([
+          0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 2, 3, 4,
+        ]),
+      );
+      expect(str.toSimpleValue()).toEqual("This is a string. It's a simple string...");
+    });
+
+    it("correctly converts primitive types to inner nested types", () => {
+      const vector = MoveVector.U8([1, 2, 3]);
+      const option = MoveOption.U8(1);
+      expect(vector.toSimpleValue()).toEqual([1, 2, 3]);
+      expect(option.toSimpleValue()).toEqual([1]);
```

Comment:
This should be just `1`, I think, or we need a better representation of Option like in Rust...

Reply:
ah I see what you mean better now- I was just trying to output what it was before with the previous remote ABI, but if that's not what makes sense it doesn't have to be

I do have an `unwrap()` function and it will return `1` but that will throw if it's undefined/null